FD limit at: 65535
zookeeper error on zoo_create, path /rtb-test/serviceClass/adServer/adserver: node exists

--------------------------[Exception thrown]---------------------------
time:   2015-06-06T06:43:19Z
type:   ML::Exception
pid:    5597; tid: 5597
what:   Zookeeper error on zoo_create, path /rtb-test/serviceClass/adServer/adserver: node exists
stack:
00: 0x0x7f365f986ecb at __cxa_throw + 0xbb in /home/ubuntu/rtbkit/build/x86_64/bin/../bin/libexception_hook.so + 0xecb/home/ubuntu/rtbkit/build/x86_64/bin/../bin/libexception_hook.so(__cxa_throw+0xbb) [0x7f365f986ecb]
01: 0x0x7f365ede0e63 at Datacratic::ZookeeperConnection::checkRes(int, int&, char const*, char const*) + 0xe3 in /home/ubuntu/rtbkit/build/x86_64/bin/../bin/libservices.so + 0x139e63/home/ubuntu/rtbkit/build/x86_64/bin/../bin/libservices.so(_ZN10Datacratic19ZookeeperConnection8checkResEiRiPKcS3_+0xe3) [0x7f365ede0e63]
02: 0x0x7f365ede0a61 at Datacratic::ZookeeperConnection::createNode(std::string const&, std::string const&, bool, bool, bool, bool) + 0x441 in /home/ubuntu/rtbkit/build/x86_64/bin/../bin/libservices.so + 0x139a61/home/ubuntu/rtbkit/build/x86_64/bin/../bin/libservices.so(_ZN10Datacratic19ZookeeperConnection10createNodeERKSsS2_bbbb+0x441) [0x7f365ede0a61]
03: 0x0x7f365ed8da85 at Datacratic::ZookeeperConfigurationService::setUnique(std::string const&, Json::Value const&) + 0x285 in /home/ubuntu/rtbkit/build/x86_64/bin/../bin/libservices.so + 0xe6a85/home/ubuntu/rtbkit/build/x86_64/bin/../bin/libservices.so(_ZN10Datacratic29ZookeeperConfigurationService9setUniqueERKSsRKN4Json5ValueE+0x285) [0x7f365ed8da85]
04: 0x0x7f365ed784d4 at Datacratic::ServiceBase::registerServiceProvider(std::string const&, std::vector<std::string, std::allocator<std::string> > const&) + 0x1c4 in /home/ubuntu/rtbkit/build/x86_64/bin/../bin/libservices.so + 0xd14d4/home/ubuntu/rtbkit/build/x86_64/bin/../bin/libservices.so(_ZN10Datacratic11ServiceBase23registerServiceProviderERKSsRKSt6vectorISsSaISsEE+0x1c4) [0x7f365ed784d4]
05: 0x0x7f365f2f6735 at RTBKIT::AdServerConnector::init(std::shared_ptr<Datacratic::ConfigurationService>) + 0xa5 in /home/ubuntu/rtbkit/build/x86_64/bin/../bin/libadserver_connector.so + 0x1f735/home/ubuntu/rtbkit/build/x86_64/bin/../bin/libadserver_connector.so(_ZN6RTBKIT17AdServerConnector4initESt10shared_ptrIN10Datacratic20ConfigurationServiceEE+0xa5) [0x7f365f2f6735]
06: 0x0x7f365f30582d at RTBKIT::HttpAdServerConnector::init(std::shared_ptr<Datacratic::ConfigurationService> const&) + 0x5d in /home/ubuntu/rtbkit/build/x86_64/bin/../bin/libadserver_connector.so + 0x2e82d/home/ubuntu/rtbkit/build/x86_64/bin/../bin/libadserver_connector.so(_ZN6RTBKIT21HttpAdServerConnector4initERKSt10shared_ptrIN10Datacratic20ConfigurationServiceEE+0x5d) [0x7f365f30582d]
07: 0x0x7f365132c828 at RTBKIT::MockAdServerConnector::init(int, int) + 0x68 in /home/ubuntu/rtbkit/build/x86_64/bin/../bin/../bin/libmock_adserver.so + 0x1e828/home/ubuntu/rtbkit/build/x86_64/bin/../bin/../bin/libmock_adserver.so(_ZN6RTBKIT21MockAdServerConnector4initEii+0x68) [0x7f365132c828]
08: 0x0x7f365132dce3 in /home/ubuntu/rtbkit/build/x86_64/bin/../bin/../bin/libmock_adserver.so + 0x1fce3/home/ubuntu/rtbkit/build/x86_64/bin/../bin/../bin/libmock_adserver.so(+0x1fce3) [0x7f365132dce3]
09: 0x0x7f365f2f7ff7 at RTBKIT::AdServerConnector::create(std::string const&, std::shared_ptr<Datacratic::ServiceProxies> const&, Json::Value const&) + 0x467 in /home/ubuntu/rtbkit/build/x86_64/bin/../bin/libadserver_connector.so + 0x20ff7/home/ubuntu/rtbkit/build/x86_64/bin/../bin/libadserver_connector.so(_ZN6RTBKIT17AdServerConnector6createERKSsRKSt10shared_ptrIN10Datacratic14ServiceProxiesEERKN4Json5ValueE+0x467) [0x7f365f2f7ff7]
10: 0x0x415b51 at main + 0x531 in build/x86_64/bin/adserver_runner + 0x15b51build/x86_64/bin/adserver_runner(main+0x531) [0x415b51]
11: 0x0x7f365e0f876d at __libc_start_main + 0xed in /lib/x86_64-linux-gnu/libc.so.6 + 0x2176d/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xed) [0x7f365e0f876d]
12: 0x0x416081 in build/x86_64/bin/adserver_runner + 0x16081build/x86_64/bin/adserver_runner() [0x416081]

terminate called after throwing an instance of 'ML::Exception'
  what():  Zookeeper error on zoo_create, path /rtb-test/serviceClass/adServer/adserver: node exists
